client/blocks/video-editor/style.scss

```diff
@@ -0,0 +1,90 @@
+.video-editor {
+	position: relative;
+	width: 100%;
+	height: 100%;
+}
+
+.editor-media-modal .video-editor .notice {
```

Comment:
I think it needs to be this way in order to be able to beat [this rule](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/media-modal/index.scss#L139).